### PR TITLE
Made TLS configurable

### PR DESCRIPTION
**Pull request by morses-code**
Making TLS configurable to allow developers to run against local instance of redis.

Run redis locally in docker and run application. 
Check http://localhost:24400/health to see redis connecting.
`$ make debug`
`$ docker run -p 6379:6379 --name redis -d redis:6.0 redis-server --requirepass "default"`

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ type Config struct {
 	ElasticachePassword        string        `envconfig:"ELASTICACHE_PASSWORD"				json:"-"`
 	ElasticacheDatabase        int           `envconfig:"ELASTICACHE_DATABASE"`
 	ElasticacheTTL             time.Duration `envconfig:"ELASTICACHE_TTL"`
+	EnableTLSConfig            bool          `envconfig:"ENABLE_TLS_CONFIG"`
```
Comment (janderson2) [inline config/config.go]: It might be worth making this variable a bit more clear that it is for redis by stating it in the variable names.

**Review by janderson2 — COMMENTED** [review]
One minor comment about the variable name and also the variable should be added to the README.

File: config/config.go (hunk 1 above)
Comment (morses-code) [reply config/config.go]: Fair enough, updated the variable name now. `EnableRedisTLSConfig` & `ENABLE_REDIS_TLS_CONFIG`

**Review by janderson2 — APPROVED** [review]

**Review by janderson2 — CHANGES_REQUESTED** [review]
Sorry, actually this new config still needs to be added to the README.

**Review by janderson2 — APPROVED** [review]

**Pull request merged**